ported/libibverbs/rc_pingpong.c

```diff
@@ -441,9 +448,15 @@ int main(int argc, char *argv[])
 
 		switch (c) {
 		case 'S':
-			size = strtol(optarg, NULL, 0);
+			errno  = 0;
+			temp = strtol(optarg, &ptr, 10);
+                        if (ptr == optarg || *ptr != '\0' || errno == ERANGE || temp < INT_MIN || temp > INT_MAX) {
+                                fprintf(stderr, "Cannot convert from string to int\n");
+				rc = 1;
```

Comment:
I suppose this is a moot point since none of the values that `strtol` can return on error are acceptable sizes, but the advice that the POSIX standard gives on this *is* to set errno to 0 before the call and check the value of errno after the call.